vendor.conf

```diff
@@ -43,8 +43,8 @@ github.com/syndtr/gocapability db04d3cc01c8b54962a58ec7e491717d06cfcc16
 github.com/gotestyourself/gotestyourself 44dbf532bbf5767611f6f2a61bded572e337010a
 github.com/google/go-cmp v0.1.0
 
-# cri dependencies
-github.com/containerd/cri b68fb075d49aa1c2885f45f2467142666c244f4a
+# #2135: cri is temporarily forked because of circular dependency. will be fixed immediately in a follow-up PR.
+github.com/containerd/cri 6e975823be192ad19f2ce7afcf6c57b88a991c30 https://github.com/AkihiroSuda/cri-containerd.git
```

Comment:
@AkihiroSuda I spotted this one, but couldn't fine an associated PR in https://github.com/containerd/cri - are you working on this?

Reply:
oh! never mind; I see that this was _just_ merged; I looked at the commit-date.